app/main.py

```diff
@@ -16,4 +16,12 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films,
+             profile_info, marks, collection_of_coins]
+sorted_variables = {"mutable": [], "immutable": []}
+
+for i in variables:
+    if isinstance(i, (list, dict, set)):
+        sorted_variables["mutable"].append(i)
+    else:
+        sorted_variables["immutable"].append(i)
```

Comment:
just place  required variables into the related lists

Reply:
> just place required variables into the related lists

Didn't understand what's wrong? The function works!